contributors/devel/godep.md

```diff
@@ -69,24 +69,19 @@ godep get $DEP/...
 ./hack/godep-save.sh
 ```
 
-To update an existing dependency is a bit more complicated.  Godep has an
-`update` command, but none of us can figure out how to actually make it work.
-Instead, this procedure seems to work reliably:
+To update an existing dependency:
 
 ```sh
-cd $KPATH/src/k8s.io/kubernetes
 DEP=example.com/path/to/dependency
 # NB: For the next step, $DEP is assumed be the repo root.  If it is actually a
 # subdir of the repo, use the repo root here.  This is required to keep godep
 # from getting angry because `godep restore` left the tree in a "detached head"
 # state.
-rm -rf $KPATH/src/$DEP # repo root
-godep get $DEP/...
-# Change code in Kubernetes, if necessary.
-rm -rf Godeps
-rm -rf vendor
+cd $KPATH/src/$DEP # repo root
+git checkout <$REVISION/$TAG/$BRANCH>
+cd $KPATH/src/k8s.io/kubernetes
+godep update $DEP/...
```

Comment:
Why do we need this update call after all? Does it speed up the following godep-save? Can we leave vendor/ and Godeps/Godeps.json when running godep-save.sh?